sources/router.ts

```diff
@@ -268,11 +268,6 @@ export function createRouter(
         asyncHandler(authenticatedOnlyMiddleware),
         asyncHandler(eventRelationshipController.updateAttendeeRole),
     );
-    router.delete(
-        "/events/:eventId/attendees/:userId",
-        asyncHandler(authenticatedOnlyMiddleware),
-        asyncHandler(eventRelationshipController.deleteAttendee),
-    );
```

Comment:
Removed because removing someone from an event but not from the calendar is not something we want